modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -154,6 +136,40 @@ The policy must be in the Stable state to upload the implementation file in a la
 
 To implement the policy so that it is available at runtime, follow the steps in xref:add-policy-implementation-files-to-published-policy-definition.adoc[Add Policy Implementation Files to Published Policy Definition].
 
+== Create a GraphQL API
+A GraphQL API is a specification of a service that defines a schema, which is the collection of object definitions. These objects are used when interacting with the API, either as input data or as part of the API responses. The GraphQL API allows you to query the API instead of accessing an operation that returns a fixed definition of data.
```

Comment:
@patricio78 please review for correctness. 
@JennyHajee please feel free to word smith it or ignore the changes if you feel the gist is captured above.

Recommended replacement:
A GraphQL API specification is a collection of object types and the operations. Object types is a collection of fields, with each field having its own type. GraphQL operations are used to interact with the API. Unlike in a REST API, where the request and response body is fixed, operations in GraphQL API don't have a fixed request and response as clients can execute the GraphQL operation to fetch just the fields needed for their use case in the response.

Reply:
The description seems ok. However we noted that for other API types there's no explanation about the spec per-se, but more a brief description of how to catalog them. That is why we used a very brief summary.

Reply:
I see. I would recommend the following cosmetic updates to the statement for accuracy. 
A GraphQL API schema is a collection of operations and object types, which are composed of fields. The GraphQL API allows a client to fetch just the object types and fields needed instead of a fixed response like in a REST API.

Reply:
sounds good @IsaacEldridge.

Reply:
done